--- include/tscpp/util/TextView.h	2022-02-16 10:57:40.674392744 -0800
+++ include/tscpp/util/TextView.h.new	2022-02-16 10:57:56.696394597 -0800
@@ -32,6 +32,7 @@
 #include <algorithm>
 #include <string>
 #include <string_view>
+#include <limits>
 
 /** Compare views with ordering, ignoring case.
  *
--- include/tscore/BufferWriterForward.h	2022-02-16 10:12:58.479082634 -0800
+++ include/tscore/BufferWriterForward.h.new	2022-02-16 10:12:39.591080450 -0800
@@ -28,6 +28,7 @@
 #include <cstring>
 #include <vector>
 #include <map>
+#include <limits>
 
 #include "tscpp/util/TextView.h"
 #include "tscore/ink_assert.h"
--- include/tscore/Extendible.h	2020-09-05 18:13:58.000000000 -0700
+++ include/tscore/Extendible.h.new	2022-02-16 11:55:08.735791403 -0800
@@ -44,6 +44,7 @@
 #include <typeinfo>
 #include <type_traits>
 #include <unordered_map>
+#include <limits>
 
 #include "tscore/AtomicBit.h"
 #include "tscore/ink_assert.h"
